powerscore-entryboard/src/components/Entryboard.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+import { reactive, ref, onMounted } from 'vue';
+import axios from 'axios'
+import LockerRoomIdentifier from './LockerRoomIdentifier.vue';
+import TeamInfo from './TeamInfo.vue';
+import placeholderImage from "@/assets/img/hockeyClipart.jpg";
+
+interface Team {
+  id: number
+  name: string,
+  image?: HTMLImageElement
+}
+
+function getPlaceholderImage(): HTMLImageElement { // TODO: get images working
+  var i = new Image();
+  i.alt = "team logo";
+  i.src = placeholderImage;
+  i.className = "team-image";
+  return i;
+}
+
+const teams = ref([])
+
+async function refreshTeams() {
+  // TODO: Authentication
+  // TODO: Do this on startup and/or periodically
+  await axios
+    .get("http://localhost:3001/teams") // API endpoint here TODO: make this externally configurable somehow
+    .then(response => {
+        console.debug(response)
+        teams.value = response.data
+    })
+}
+
+
+onMounted( () => {
```

Comment:
Can `onMounted` not take an `async` function? Like, can we mark this lambda as `async` and then call `await` on `refreshTeams`?